### PR TITLE
Feature: social login

### PR DESCRIPTION
**Pull request by massiresogore**
Here can write some detail ...

### CODE REVIEW

File: file3.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello
```
Comment (msrmossaka) [inline file3.txt]: This line is goot, but it is better to capitalize H.

**Review by msrmossaka — CHANGES_REQUESTED** [review]
please make the changes and do another push

**Review by massiresogore — COMMENTED** [review]
Everything look perfect, let's go ahead whith mergin

**Review by msrmossaka — APPROVED** [review]
Everything looks good, let's go ahead to merge

**Pull request merged**